server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -372,7 +372,8 @@ public static MetadataVersion fromFeatureLevel(short version) {
 
     // Testing only
     public static MetadataVersion latestTesting() {
-        return VERSIONS[VERSIONS.length - 1];
+        // There is no separate testing version because all features are currently stable.
+        return LATEST_PRODUCTION;
```

Comment:
Perhasp we could add `IBP_4_3_IV0` to align the code with other 4.x branches? This would also make the patch easier to cherry-pick to trunk

Reply:
Well, I did consider this, but there's no need for `IBP_4_3_IV0` because there are no unstable features.

Reply:
I prefer #20977 because it helps to avoid the awkward `disabled` tag :smiley: